sys/hacl/build.rs

```diff
@@ -171,6 +187,10 @@ fn compile_files(
         build.include(include);
     }
     build.opt_level(3);
+    build.static_crt(true);
+    if let Some(target) = platform.target {
+        build.flag(&format!("--target={target}"));
+    }
```

Comment:
I'm assuming this flag is to enable cross-compile. We may need to check for `msvc` to avoid using `--target` flag there.

Reply:
Actually this is done automatically. This is a leftover. I'll delete it.